sagemaker-python-sdk/pytorch_batch_inference/sagemaker_batch_inference_torchserve.ipynb

```diff
@@ -0,0 +1,463 @@
+{
```

Comment:
@nskool we can add a paragraph at the beginning similar, " This notebook  demonstrates how to do batch processing on Sagemaker using the dynamic batching feature provided in Torchserve. This example uses a HuggingFace Bert based model for sequence classification.

Reply:
Done